### PR TITLE
DBSync update 2022-09-19

### PR DESCRIPTION
**Pull request by kderme**

### CODE REVIEW

File: posts/2022-09-19-db-sync.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# DBSync Update
```
Comment (dnadales) [inline posts/2022-09-19-db-sync.markdown:1]: I wonder if it'd make sense to wrap lines to 80 characters.

**Review by dnadales — APPROVED** [review]
Great to see db-sync updates here as well!

File: posts/2022-09-19-db-sync.markdown (hunk 1 above)
Comment (JaredCorduan) [reply posts/2022-09-19-db-sync.markdown:1]: I've been (quite arbitrarily!) trying to keep lines to 100 characters, except when I have a link that's really long (or maybe it's in the CIPs where I hit that..)

**Review by JaredCorduan — APPROVED** [review]

File: posts/2022-09-19-db-sync.markdown (hunk 1 above)
Comment (kderme) [reply posts/2022-09-19-db-sync.markdown:1]: I changed this to max 100 chars/line, except for big links.

**Pull request merged**